insights/util/autology/datasources.py

```diff
@@ -7,11 +7,11 @@
 import inspect
 import insights.core.spec_factory
 import insights.specs
+
+from insights.core.exceptions import AutologyError
 from insights.specs.default import DefaultSpecs as InDefaultSpecs
 from insights.specs.insights_archive import InsightsArchiveSpecs as InArchiveSpecs
 from insights.specs.sos_archive import SosSpecs as InSosSpecs
-from . import AutologyError
```

Comment:
@ryan-blakley I don't think its necessary to move this exception.  Since this is a utility module, its not used by core for normal processing.